Dev_1.java

```diff
@@ -1,11 +1,15 @@
 public class Dev_1 {
   public static void main(String[] args) {
     for (int i = 0; i <= 100; i++) {
-      if (i % 3 == 0) {
-        System.out.println("3*" + i / 3);
+      if (i % 3 == 0 && i % 5 != 0) {
+        System.out.println("Tutti");
+      } else if (i % 5 == 0 && i % 3 != 0) {
+        System.out.println("Frutti");
+      } else if (i % 3 == 0 && i % 5 == 0) {
```

Comment:
If you re-order all these ifs you can avoid not-equal comparison - it'll make code better